src/store/validation/index.ts

```diff
@@ -97,7 +97,7 @@ export const actions: ActionTree<IValidationState, IRootState> = {
       ...status,
       ajvErrors: getLocalizedAjvErrors(key, status.ajvErrors),
     })
-    if (status.ajvErrors === null) {
+    if (status.ajvErrors === null || status.ajvErrors.length === 0) {
```

Comment:
We could alternatively return null from `errors.list()` to match the ajv interface
(and change logic for `validationResults.ajvErrors`)